runbotics/runbotics-orchestrator-ui/src/contentful/marketplace-main/types.ts

```diff
@@ -0,0 +1,52 @@
+import { getMainPage } from '#contentful/blog-main';
+import { FetchContentfulResponse, Industry, MarketplaceOffer, Tag } from '#contentful/common';
+
+interface AllOffersCollection {
```

Comment:
FYI If you consider using some types between runbotics-orchestrator-ui and runbotics-scheduler you can place them in runbotics-common build it (rushx build) and use them across the whole monorepo)